activesupport/lib/active_support/core_ext/date_time/calculations.rb

```diff
@@ -139,14 +146,32 @@ def end_of_minute
   end
   alias :at_end_of_minute :end_of_minute
 
-  # Adjusts DateTime to UTC by adding its offset value; offset is set to 0.
+  # Returns a <tt>Time</tt> instance of the simultaneous time in the system timezone.
```

Comment:
@matt-glover hmm, definitely a change here isn't great - it was never the intention when I originally wrote this code for it to be backported so I may have overlooked something. We're currently tweaking this code anyway because of #28147 so I'll look at it as part of that.